components/core/src/LogTypeDictionaryEntry.hpp

```diff
@@ -49,15 +50,20 @@ class LogTypeDictionaryEntry : public DictionaryEntry<logtype_dictionary_id_t> {
 
     // Methods
     /**
-     * Adds a non-double variable delimiter to the given logtype
+     * Adds a dictionary variable delimiter to the given logtype
      * @param logtype
      */
-    static void add_non_double_var (std::string& logtype) { logtype += (char)VarDelim::NonDouble; }
+    static void add_dict_var (std::string& logtype) { logtype += (char)VarDelim::Dictionary; }
```

Comment:
These uses of `VarDelim` should also be replaced with `static_cast<char>` or `enum_to_underlying_type`, whatever you end up deciding.